tools/wpt/wpt.py

```diff
@@ -77,15 +80,19 @@ def import_command(prog, command, props):
     return script, parser
 
 
-def setup_virtualenv(path, props):
+def setup_virtualenv(path, skip_venv_setup, props):
+    if skip_venv_setup and path is None:
+        raise ValueError("Must set --venv when --skip-venv-setup is used")
+    should_skip_setup = path is not None and skip_venv_setup
     if path is None:
         path = os.path.join(wpt_root, "_venv")
-    venv = virtualenv.Virtualenv(path)
-    venv.start()
-    for name in props["install"]:
-        venv.install(name)
-    for path in props["requirements"]:
-        venv.install_requirements(path)
+    venv = virtualenv.Virtualenv(path, should_skip_setup)
+    if not should_skip_setup:
+        venv.start()
```

Comment:
It'd be one or the other, right? If we remove the changes in `virtualenv.py` then we need this function to skip calling `start` and `install`. If we keep the changes in `virtualenv.py` then this function can just call all these methods since they'd be no-ops.

I agree we probably don't need both, so I'll change this based on the other comment above.

Reply:
Done